### PR TITLE
Improvements according to PR #2 comments

### PR DESCRIPTION
**Pull request by mustafa-cosar**
I had published the `v0.0.1` so I also updated the version

### CODE REVIEW

File: src/int64_util.js [hunk 1]
```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-// This file is copied from the Thrift project under Apache 2.0 license
-// There are some changes to variable names and styling
-// The other changes are documented before the changes
+// This file is copied from the Thrift project under Apache 2.0 license. There are some changes to variable names and
```
Comment (bgedik) [inline src/int64_util.js:20]: the Apache 2.0 license

File: src/int64_util.js [hunk 2]
```diff
@@ -17,9 +17,8 @@
...
+// styling. The other changes are documented before the changes
```
Comment (bgedik) [inline src/int64_util.js:21]: Missing . at the end.
Also, I was not able to make sense of the last sentence

File: src/int64_util.js [hunk 3]
```diff
@@ -40,40 +39,37 @@ Int64Util.toDecimalString = function (i64)
         // The magnitude is small enough.
         return i64.toString();
     }
-    else
+    var negative = i64Buffer[i64Offset] & 0x80;
```
Comment (bgedik) [inline src/int64_util.js:42]: magic constant 0x80

File: src/int64_util.js [hunk 4]
```diff
@@ -40,40 +39,37 @@ Int64Util.toDecimalString = function (i64)
...
+    if (negative)
     {
-        var negative = i64Buffer[i64Offset] & 0x80;
-        if (negative)
+        // 2's complement
+        var incremented = false;
+        var buffer = new Buffer(8);
```
Comment (bgedik) [inline src/int64_util.js:47]: magic constants 8, 7

File: src/int64_util.js [hunk 5]
```diff
@@ -40,40 +39,37 @@ Int64Util.toDecimalString = function (i64)
...
-            // 2's complement
-            var incremented = false;
-            var buffer = new Buffer(8);
-            for (var i = 7; i >= 0; --i)
-            {
-                buffer[i] = (~i64Buffer[i64Offset + i] + (incremented ? 0 : 1)) & 0xff;
-                incremented |= i64Buffer[i64Offset + i];
-            }
-            i64Buffer = buffer;
+            buffer[i] = (~i64Buffer[i64Offset + i] + (incremented ? 0 : 1)) & 0xff;
```
Comment (bgedik) [inline src/int64_util.js:50]: magic constant 0xff

File: src/int64_util.js [hunk 6]
```diff
@@ -40,40 +39,37 @@ Int64Util.toDecimalString = function (i64)
...
-        {
-            low = String(low % POW10_11).slice(-11);
-            return (negative ? '-' : '') + low;
-        }
-        // Make it exactly 11 with leading zeros.
-        low = ('00000000000' + String(low % POW10_11)).slice(-11);
-        return (negative ? '-' : '') + String(high) + low;
+        i64Buffer = buffer;
+    }
+    var high2 = i64Buffer[i64Offset + 1] + (i64Buffer[i64Offset] << 8);
```
Comment (bgedik) [inline src/int64_util.js:55]: various magic constants

File: src/int64_util.js [hunk 7]
```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-// This file is copied from the Thrift project under Apache 2.0 license
-// There are some changes to variable names and styling
-// The other changes are documented before the changes
```
Comment (bgedik) [inline src/int64_util.js:22]: Are there tests for these utilities?

**Review by bgedik — APPROVED** [review]

File: src/int64_util.js (hunk 2 above)
Comment (mustafa-cosar) [reply src/int64_util.js:21]: It was my understanding that I need to document all alterations on this file so I added a comment here and before the changes themselves. But I have checked again and it seems like there is no need to document every single change.

File: src/int64_util.js (hunk 7 above)
Comment (mustafa-cosar) [reply src/int64_util.js:22]: Yes, there are

File: src/int64_util.js (hunk 1 above)
Comment (mustafa-cosar) [reply src/int64_util.js:20]: fixed

File: src/int64_util.js (hunk 3 above)
Comment (mustafa-cosar) [reply src/int64_util.js:42]: fixed

File: src/int64_util.js (hunk 4 above)
Comment (mustafa-cosar) [reply src/int64_util.js:47]: fixed

File: src/int64_util.js (hunk 5 above)
Comment (mustafa-cosar) [reply src/int64_util.js:50]: fixed

File: src/int64_util.js (hunk 6 above)
Comment (mustafa-cosar) [reply src/int64_util.js:55]: fixed

**Pull request merged**